src/map/script.c

```diff
@@ -8616,6 +8616,58 @@ static BUILDIN(delitem2)
 	return false;
 }
 
+/**
+ * Deletes item at given index.
+ * delitem(<index>{, <account id>});
+ */
+static BUILDIN(delitemidx)
+{
+	struct map_session_data *sd;
+
+	if (script_hasdata(st, 3)) {
+		if ((sd = script->id2sd(st, script_getnum(st, 3))) == NULL) {
+			script_pushint(st, 0);
+			st->state = END;
```

Comment:
the script already halt by `st->state = END`, so no point to `script_pushint` anymore
in fact, nobody use `*delitem` to return 0 in script, are they ?
```c
if ( delitem(Apple,1) == false )
mes "fail to delete item";
```
do you ever see anybody doing this ? no ... right ?
because the script already halt